packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -209,7 +219,7 @@ interface UserMenuProps {
 
   /**
    * MongoDB platform that is currently active.
-   * Possible values: ['account', 'cloud',  'support', 'university']
+   * Possible values: ['account', 'cloud',  'support', 'university', 'devHub']
```

Comment:
this comment seems redundant with the type's definition. we can remove it so we don't have to keep it in sync

Reply:
👍